src/Decoder/YamlDecoder.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace League\JsonReference\Decoder;
+
+use League\JsonReference\DecoderInterface;
+use League\JsonReference\DecodingException;
+
+use Symfony\Component\Yaml\Exception\ParseException;
+use Symfony\Component\Yaml\Yaml;
```

Comment:
This dependency should either be added to `require` or `suggest`.  I would prefer `suggest` but that isn't possible if you are registering the YAML loader by default.